hs_file_types/models/timeseries.py

```diff
@@ -518,6 +525,12 @@ def remove_term(term):
                     graph.add((unit_node, HSTERMS.UnitsAbbreviation, units_abbreviation))
                     graph.remove((result_node, HSTERMS.UnitsAbbreviation, units_abbreviation))
 
+        if self.abstract:
+            # abstract is all by itself on this model, won't get picked up by rdf/xml creation automatically
```

Comment:
Does the comment above apply here?

Reply:
I think it's helpful to highlight why abstract has to be manually accounted for.